nginx-controller/controller/controller.go

```diff
@@ -1204,6 +1204,7 @@ func (lbc *LoadBalancerController) getMinionsForMaster(master *nginx.IngressEx)
 	}
 
 	var minions []*nginx.IngressEx
+IngressLoop:
```

Comment:
Instead of using the label we can also set a boolean flag to continue if a certain annotation is found or if there are multiple paths, but using a label is quicker since we won't have to wait for the loop to end.